src/modules/wallhack.rs

```diff
@@ -0,0 +1,74 @@
+//! `bxt_wallhack`
+
+use super::Module;
+use crate::gl;
+use crate::hooks::engine;
+use crate::modules::cvars::{self, CVar};
+use crate::utils::*;
+
+pub struct Wallhack;
+impl Module for Wallhack {
+    fn name(&self) -> &'static str {
+        "bxt_wallhack"
+    }
+    fn cvars(&self) -> &'static [&'static CVar] {
+        static CVARS: &[&CVar] = &[&BXT_WALLHACK, &BXT_WALLHACK_ADDITIVE, &BXT_WALLHACK_ALPHA];
+        &CVARS
+    }
+
+    fn is_enabled(&self, marker: MainThreadMarker) -> bool {
+        engine::R_DrawSequentialPoly.is_set(marker) && cvars::CVars.is_enabled(marker)
+    }
+}
+
+static BXT_WALLHACK: CVar = CVar::new(b"bxt_wallhack\0", b"0\0");
+static BXT_WALLHACK_ADDITIVE: CVar = CVar::new(b"bxt_wallhack_additive\0", b"0\0");
+static BXT_WALLHACK_ALPHA: CVar = CVar::new(b"bxt_wallhack_alpha\0", b"0.6\0");
```

Comment:
Does anyone ever change this? Maybe we can get rid of these?

Reply:
Alpha could be hardcoded in ranging of 0.5-0.75 values, but please don't remove **bxt_wallhack_additive**.

Reply:
Maybe hardcode alpha to 0.6 and remove cvar then?

Reply:
Sounds good to me, done.